prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
According to the package.json you arent installing prettier (but you are using eslint), so why the config file for this and not for eslint?